### ISSUE TITLE
Prevent the generation of empty text blocks.

### ISSUE THREAD

**Issue by dobeerman** [matteodf/html-to-slack#9]
**Describe the bug**
In some cases empty text blocks are generated.
`[ERROR]  bolt-app must be more than 0 characters [json-pointer:/blocks/3/elements/0/elements/0/elements/9/text]`

**To Reproduce**
```json
{
  "id": "tag:github.com,2008:Repository/105808767/v2.171.1",
  "updated": "2024-11-27T19:13:03Z",
  "link": {
    "attributes": {
      "rel": "alternate",
      "type": "text/html",
      "href": "https://github.com/aws/aws-cdk/releases/tag/v2.171.1"
    }
  },
  "title": "v2.171.1",
  "content": {
    "text": "<h3>Bug Fixes</h3>\\n<ul>\\n<li><strong>cli:</strong> lambda hotswap fails if <code>lambda:GetFunctionConfiguration</code> action is not allowed  (<a href=\\\"https://github.com/aws/aws-cdk/issues/32301\\\" data-hovercard-type=\\\"pull_request\\\" data-hovercard-url=\\\"/aws/aws-cdk/pull/32301/hovercard\\\">#32301</a>) (<a href=\\\"https://github.com/aws/aws-cdk/commit/a073e9302dbd4213275e99c86476ab8152af7caf\\\">a073e93</a>), closes <a href=\\\"https://github.com/aws//github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdatedV2.ts/issues/L10\\\">/github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdatedV2.ts#L10</a> <a href=\\\"https://github.com/aws//github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdated.ts/issues/L13\\\">/github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdated.ts#L13</a></li>\\n</ul>\\n<hr>\\n<h2>Alpha modules (2.171.1-alpha.0)</h2>",
    "attributes": { "type": "html" }
  },
  "author": { "name": "aws-cdk-automation" },
  "media:thumbnail": {
    "attributes": {
      "height": "30",
      "width": "30",
      "url": "https://avatars.githubusercontent.com/u/43080478?s=60&v=4"
    }
  }
}
```

**Expected behavior**
No blocks with empty text generated.
 
**Result**
```json
[
  {
    "type": "header",
    "text": {
      "type": "plain_text",
      "text": ":rocket: Release notes from aws-cdk - v2.171.1 (2024-11-27)",
      "emoji": true
    }
  },
  { "type": "divider" },
  { "type": "header", "text": { "type": "plain_text", "text": "Bug Fixes" } },
  {
    "type": "rich_text",
    "elements": [
      {
        "type": "rich_text_list",
        "style": "bullet",
        "elements": [
          {
            "type": "rich_text_section",
            "elements": [
              { "type": "text", "text": "cli:", "style": { "bold": true } },
              { "type": "text", "text": " lambda hotswap fails if " },
              {
                "type": "text",
                "text": "lambda:GetFunctionConfiguration",
                "style": { "code": true }
              },
              { "type": "text", "text": " action is not allowed  (" },
              {
                "type": "link",
                "url": "https://github.com/aws/aws-cdk/issues/32301",
                "text": "#32301"
              },
              { "type": "text", "text": ") (" },
              {
                "type": "link",
                "url": "https://github.com/aws/aws-cdk/commit/a073e9302dbd4213275e99c86476ab8152af7caf",
                "text": "a073e93"
              },
              { "type": "text", "text": "), closes " },
              {
                "type": "link",
                "url": "https://github.com/aws//github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdatedV2.ts/issues/L10",
                "text": "/github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdatedV2.ts#L10"
              },
              { "type": "text", "text": "" }, <<<<<<<<<<<
              {
                "type": "link",
                "url": "https://github.com/aws//github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdated.ts/issues/L13",
                "text": "/github.com/aws/aws-sdk-js-v3/blob/main/clients/client-lambda/src/waiters/waitForFunctionUpdated.ts#L13"
              }
            ]
          }
        ],
        "indent": 0
      }
    ]
  },
  {
    "type": "header",
    "text": { "type": "plain_text", "text": "Alpha modules (2.171.1-alpha.0)" }
  },
  { "type": "divider" },
  {
    "type": "context",
    "elements": [
      {
        "type": "mrkdwn",
        "text": ":busts_in_silhouette: aws-cdk-automation, <https://github.com/aws/aws-cdk/releases/tag/v2.171.1|Open v2.171.1 release page>"
      }
    ]
  }
]

```

**Comment by dobeerman** [comment]
[this line](https://github.com/matteodf/html-to-slack/blob/main/src/parsers/index.ts#L253) is the problem. maybe another one, but that was the easiest workaround to remove the last replace

**Comment by matteodf** [comment]
Hi @dobeerman ,

Sorry I could not answer you sooner. Actually, that `.replace()` was no longer needed: it probably solved problems initially that were solved in other ways in later commits. I removed it in version 0.3.6, which should fix the problem.

Thanks for the report, and sorry again for the delay, it's a really busy time for me. If you have other suggestions or experience other problems, feel free to report them in a new issue!
